### ISSUE TITLE
Learner redirected to 'Select/Change facility' again after signing in with the new credentials after the 'Merge account'

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#12762]
## Observed behavior
Discovered while testing https://github.com/learningequality/kolibri/pull/12761.

@rtibbles suggested that the error might be generated when trying to access the "login in as merged user" endpoint - probably a 403 error... 

## Errors and logs
…

## Expected behavior
More seamless experience after the merge: if the authentication is necessary, user should land on the Learn home page.

## User-facing consequences
Confusion, was the merge completed as requested, or not...? 

## Steps to reproduce
(1:48 in the video below)

https://github.com/user-attachments/assets/001d7622-ec88-4602-a209-fe0760842044

## Context

 * Kolibri version 0.17.3b0
 * Operating system: Windows 10
 * Browser: Firefox

**Comment by emily-dominguez** [comment]
hi! could i get this assigned?

**Comment by rtibbles** [comment]
Yes, @emily-dominguez - do note that effectively testing this workflow will involve running multiple Kolibri servers at once, in order to have a server to connect to, in order to merge your 'on my own setup' user into.

Please feel free to ask any questions here as you are exploring the issue.

**Comment by emily-dominguez** [comment]
i see, or actually is help still wanted here (https://github.com/learningequality/kolibri/issues/12647) ?

**Comment by rtibbles** [comment]
I commented on that issue - I don't think it's suitable for contribution at this juncture.

**Comment by rtibbles** [comment]
If this issue seems too much, I have made sure to clear assignees from other help wanted issues where there had not been contribution in a long time. Please see the help wanted issues here: https://github.com/learningequality/kolibri/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22help%20wanted%22

**Comment by emily-dominguez** [comment]
@rtibbles if this may be too much, what would be the best issues to start off with?

**Comment by rtibbles** [comment]
@emily-dominguez see the link I put above for a list of help wanted issues.

**Comment by MisRob** [comment]
If that helps, here's guidance on [Running another Kolibri instance alongside the development server](https://kolibri-dev.readthedocs.io/en/develop/howtos/another_kolibri_instance.html)

**Comment by emily-dominguez** [comment]
hi @MisRob! so running another Kolibri instance would be create an account or why exactly is another instance needed?

**Comment by emily-dominguez** [comment]
![Image](https://github.com/user-attachments/assets/885527cf-cb94-4a72-a8ef-108ac1d57710)
is it for this to be able to connect?

**Comment by MisRob** [comment]
Hi @emily-dominguez, do you have access to see the video in the issue description? It shows going through "Select facility" and then merge accounts. So the other facility is needed so you can select it and have an account to merge with.

**Comment by emily-dominguez** [comment]
hi @MisRob, I see in the video that two facilities need to be added, but I can't add a facility When I run the server I get the above image (it never connects or says error) and when I do it on the normal Kolibri (with out running the server) I get the same error). How can I get a facility to be added or is there an admin / testing login / password that I could use?  Would it be possible to message on slack to communicate more easier?

**Comment by emily-dominguez** [comment]
like I made another account through the Kolibri app and I get the following: 
If I could get any help on this, it would be greatly appreciated  as I can only work on it today until a while !

![Image](https://github.com/user-attachments/assets/e31917fb-df84-4f60-91eb-fac1a1cd10d3)

**Comment by rtibbles** [comment]
Hi @emily-dominguez I am responding here, rather than on the merged PR you commented on:

> hey @rtibbles! I'm working on this issue, and I was wondering if I could get some help understanding the flow, so it seems like a user successfully mergers an account in the file confrimMerge.vue where does this request/path get sent to? is it suppose to call the LoginMergedUserViewset(APIView)? but it's not doing so?

Yes - the user shouldn't have to sign in again after the merge has happened, so something is going wrong in the flow. 

The call to the login endpoint is meant to happen in this component: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/user_profile/assets/src/views/ChangeFacility/MergeFacility.vue#L281

So I would start by checking there - my recollection is that it is being called, but perhaps is being blocked by a permissions issue, but I may be wrong!

**Comment by emily-dominguez** [comment]
Hi @rtibbles! I see do you know how I can replicate the above issue, I'm having issues with adding in a facility per the above screen shots

**Comment by emily-dominguez** [comment]
it seems like my network doesn't connect to add in another facility

**Comment by MisRob** [comment]
Hi @emily-dominguez 

I'm sorry for the trouble. It may be best if you reported what you're experiencing with detailed steps to reproduce and as much information about your operating system, browser, etc. in a new issue. It seems like a bug that will need further investigation.

If you'd still like to contribute, I'd recommend finding 'help wanted' issue that doesn't require multiple facilities setup.

**Comment by MisRob** [comment]
Hi @emily-dominguez,  I wanted to mention that [Learning Equality will be closed from December 23 to January 5](https://github.com/learningequality/kolibri/discussions/12956).

**Comment by AlexVelezLl** [comment]
Hey @emily-dominguez, Are you stil working on this issue?
